oneBarangay/static/assets/js/volt.js

```diff
@@ -15,166 +15,166 @@
+    })
   }
 
-  [].slice.call(d.querySelectorAll("[data-background]")).map(function (el) {
-    el.style.background = "url(" + el.getAttribute("data-background") + ")";
+  [].slice.call(d.querySelectorAll('[data-background]')).map(function (el) {
+    el.style.background = 'url(' + el.getAttribute('data-background') + ')'
   });
 
-  [].slice.call(d.querySelectorAll("[data-background-lg]")).map(function (el) {
+  [].slice.call(d.querySelectorAll('[data-background-lg]')).map(function (el) {
     if (document.body.clientWidth > breakpoints.lg) {
-      el.style.background = "url(" + el.getAttribute("data-background-lg") + ")";
+      el.style.background = 'url(' + el.getAttribute('data-background-lg') + ')'
     }
   });
 
-  [].slice.call(d.querySelectorAll("[data-background-color]")).map(function (el) {
-    el.style.background = "url(" + el.getAttribute("data-background-color") + ")";
+  [].slice.call(d.querySelectorAll('[data-background-color]')).map(function (el) {
+    el.style.background = 'url(' + el.getAttribute('data-background-color') + ')'
   });
 
-  [].slice.call(d.querySelectorAll("[data-color]")).map(function (el) {
-    el.style.color = "url(" + el.getAttribute("data-color") + ")";
-  });
+  [].slice.call(d.querySelectorAll('[data-color]')).map(function (el) {
+    el.style.color = 'url(' + el.getAttribute('data-color') + ')'
+  })
 
   // Tooltips
-  const tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'));
+  const tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'))
   const tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
-    return new bootstrap.Tooltip(tooltipTriggerEl);
-  });
+    return new bootstrap.Tooltip(tooltipTriggerEl)
+  })
 
   // Popovers
-  const popoverTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="popover"]'));
+  const popoverTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="popover"]'))
   const popoverList = popoverTriggerList.map(function (popoverTriggerEl) {
-    return new bootstrap.Popover(popoverTriggerEl);
-  });
+    return new bootstrap.Popover(popoverTriggerEl)
+  })
 
   // Datepicker
-  const datepickers = [].slice.call(d.querySelectorAll("[data-datepicker]"));
+  const datepickers = [].slice.call(d.querySelectorAll('[data-datepicker]'))
   const datepickersList = datepickers.map(function (el) {
     return new Datepicker(el, {
-      buttonClass: "btn",
-    });
-  });
+      buttonClass: 'btn'
+    })
+  })
 
-  if (d.querySelector(".input-slider-container")) {
-    [].slice.call(d.querySelectorAll(".input-slider-container")).map(function (el) {
-      const slider = el.querySelector(":scope .input-slider");
-      const sliderId = slider.getAttribute("id");
-      const minValue = slider.getAttribute("data-range-value-min");
-      const maxValue = slider.getAttribute("data-range-value-max");
+  if (d.querySelector('.input-slider-container')) {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=7999497)
```suggestion
    if(d.querySelector(".input-slider-container")) {
```